views/index.pug

```diff
@@ -5,75 +5,64 @@ html(lang="en")
-              | Thanks for using Compiler Explorer
-            span.community-hide(title="Hide and never show again" aria-label="Close")
-              button.close(type="button" aria-hidden="true") &times;
-          ul.nav.navbar-nav.navbar-right
-            li: a#share(href="javascript:;") Share#[b.caret]
-            li.dropdown
-              a.dropdown-toggle(href="javascript:;" data-toggle="dropdown") Other#[b.caret]
-              ul.dropdown-menu.menu-other
-                li
-                  a(href="https://www.patreon.com/bePatron?u=3691963"
-                  title="Help Compiler Explorer - become a Patron" target="_blank" rel="noopener")
-                    img(height="20" width="20" src=require("patreon_logo.png"))
-                    | &nbsp; Become a Patron
-                if githubEnabled
-                  li
-                    a(href="https://github.com/mattgodbolt/compiler-explorer"
-                    title="View the source on GitHub" target="_blank" rel="noopener")
-                      img(height="20" width="20" src=require("GitHub-Mark-20px.png"))
-                      | &nbsp; Source on GitHub
-                li
-                  a(href="https://groups.google.com/forum/#!forum/compiler-explorer-discussion"
-                  title="Join Compiler Explorer Public Google Group" target="_blank" rel="noopener")
-                    img(height="20" width="20" src=require("Google-Groups-20px.png"))
-                    | &nbsp; Mailing list
-                li: a(href="https://github.com/mattgodbolt/compiler-explorer/wiki/Installed-libraries" target="_blank" rel="noopener") Installed libraries
-                li: a(href="https://github.com/mattgodbolt/compiler-explorer/wiki" target="_blank" rel="noopener") Wiki
-                li: a(href="https://github.com/mattgodbolt/compiler-explorer/issues" target="_blank" rel="noopener") Report an issue
-                li#thanks-to: a(href="javascript:;") Thanks to...
-                li: a(href="https://xania.org/201609/how-compiler-explorer-runs-on-amazon" target="_blank") How it works
-                li: a(href="mailto:matt@godbolt.org" target="_blank" rel="noopener noreferrer") Contact the author
-                li: a(href="https://xania.org/MattGodbolt" rel="author" target="_blank") About the author
-                li#changes: a(href="javascript:;") Changelog
-                if sharingEnabled
-                  li#socialshare
-            if policies.cookies.enabled || policies.privacy.enabled
-              li.dropdown
-                a.dropdown-toggle(href="javascript:;" data-toggle="dropdown") Policies#[b.caret]
-                ul.dropdown-menu.menu-other
-                  if policies.cookies.enabled
-                    li#cookies: a(href="javascript:;") Cookie policy
-                  if policies.privacy.enabled
-                    li#privacy: a(href="javascript:;") Privacy policy
-
+    nav.navbar.navbar-godbolt.navbar-expand-md.navbar-light.bg-light
```

Comment:
Nice! These are the "actual" bootstrap 4 changes, I take it? no more `container-fluid` nonsense? Yay

Reply:
Yes! I find it's now a bit easier to place elements as you want too